templates/hcloud-csi.yml

```diff
@@ -14,6 +14,7 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: hcloud-volumes
+  namespace: {{ .Release.Namespace }}
```

Comment:
>I add metadata "namespace"... @invidian why you not use "kube-system" namespace from original deployment yaml?

As far as I know, Helm 3 recommends to not set the namespace explicitly when possible, so created resources will use the namespace for kubeconfig context. This allows to install the chart in any namespace. And there is no specific reason for those resources to be installed in `kube-system` namespace. Any other namespace works just fine for it.

If original code comes with `namespace: kube-system`, I can either remove the line or template it to make it work in other namespace, so removing seems simpler to me, that's why it's removed. Could you please remove it as well?

As a hint for the future, for doing this updates, I use `wget` to write new upstream content into the files and then use `git restore -p` to selectively revert back the templating. This way, it's easy to pick only important upstream changes without touching the templating.

Reply:
I did this.